api/services/ocean/spark/paths/clusterConfigTemplate.yaml

```diff
@@ -0,0 +1,113 @@
+summary: Ocean Spark
+
+get:
+  summary: "Get Config Template"
+  description: >
+    Describe a config template tied to an Ocean Spark cluster.
+  operationId: "OceanSparkClusterConfigTemplateGet"
+  tags:
+    - "Ocean Spark"
+  parameters:
+    - $ref: "../../../../commons/parameters/nonRequiredAccountId.yaml"
+    - $ref: "../parameters/path/oceanSparkClusterId.yaml"
+    - $ref: "../parameters/path/oceanSparkConfigTemplateId.yaml"
+  responses:
+    200:
+      $ref: "../responses/configTemplates.yaml"
+    400:
+      description: "Bad Request"
+
+patch:
+  summary: "Patch Config Template"
+  description: >
+    Update a config template by merging the existing configuration with the one provided in the request body.
+  operationId: "OceanSparkClusterConfigTemplatePatch"
+  tags:
+    - "Ocean Spark"
+  parameters:
+    - $ref: "../../../../commons/parameters/nonRequiredAccountId.yaml"
+    - $ref: "../parameters/path/oceanSparkClusterId.yaml"
+    - $ref: "../parameters/path/oceanSparkConfigTemplateId.yaml"
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          required:
+          - displayName
+          - config
```

Comment:
@crezvoy are those two fields effectively required? What if I want to update displayName only, and vice-versa?

Reply:
Yes currently both are required. If you want to set only a new display name, you have to send the unchanged config  with it (An empty config would actually do the trick as well as it is merged with the existing one). it would indeed be more natural to make these params optionals.

Reply:
Fields will be made non required in https://spotinst.atlassian.net/browse/BGD-1654